mxcubeweb/core/components/samplechanger.py

```diff
@@ -385,6 +389,16 @@ def sync_with_crims(self):
             logging.getLogger("MX3.HWR").exception("Could not get crystal List")
             return {"xtal_list": xtal_list}
 
+    def _gripper_changed(self):
+        from mxcubeweb.routes import signals
+
+        self.app.queue.queue_clear()
+        self.app.server.emit(
+            "queue", {"Signal": "update", "message": "all"}, namespace="/hwr"
+        )
+
+        # self.app.sample_changer.get_sample_list()
+
```

Comment:
Is the import necessary? I do not see it being used. Or is it an import with side effects?

```suggestion
    def _gripper_changed(self):
        self.app.queue.queue_clear()
        self.app.server.emit(
            "queue", {"Signal": "update", "message": "all"}, namespace="/hwr"
        )
```

Reply:
:+1: Noh its not needed :). It was something from a previous solution I tried. Thanks !